### DISCUSSION TITLE
Sortind enums,types - do not preserve the scheme

### DISCUSSION THREAD

**Discussion by pronovaso** [dotansimha/graphql-code-generator#5337] [Q&A] [answered]
Hi.

If I have enum like this:

`enum State {
   KONCEPT,
   CEKA
}
`
after generate I have 
`enum State {
   CEKA,
   KONCEPT
}
`

after update all codegen dependecies

**Comment by ardatan** [comment]
I don't think it is possible to keep the order in your provided schema. We can only do alphanumeric sorting because there are too many objects created, deep-copied and more. @dotansimha What do you think?

**Reply by pronovaso** [reply]
This behaviour is same for type. Before update all dependencies, the behaviour was right. Thanks

**Comment by dotansimha** [comment, marked as answer]
This issue is being tracked in: https://github.com/dotansimha/graphql-code-generator/issues/5106
